src/care/host_ptr.h

```diff
@@ -69,12 +69,15 @@ namespace care {
                    typename std::enable_if<B, int>::type = 1>
          host_ptr<T>(host_ptr<T_non_const> const &ptr) noexcept : m_ptr(ptr.data()) {}
 
+         // TODO: We can probably just use .data in the future, but some
+         // packages that depend on CARE use an older version of CHAI.
+#if !defined(CARE_ENABLE_IMPLICIT_CONVERSIONS)
          ///
```

Comment:
I wonder if we really need both cases here? We could probably just use host_ptr(chai::ManagedArray<T> const &ptr) : m_ptr(ptr.data()) {} in both cases regardless of if implicit conversions are on and off, would work the same and be less maintenance. Just have a comment explaining why you have .data() instead

Reply:
Nevermind, missed the comment above

Reply:
So the .data method is in CHAI develop, but there is no CHAI release that contains it. That's why I'm hesitant to unconditionally use it. Most people stick to a tagged release of CHAI, in which case using the .data method will cause build failures. I do have a comment on lines 72-74 about that. But using CARE_ENABLE_IMPLICIT_CONVERSIONS is probably not the right thing to do. I may have to do something based on the version number or use the getPointer method instead since that was present in v2.1.1 (the latest release).

Reply:
I decided to use .getPointer and added a comment about updating to .data when the next release of CHAI occurs.